apps/mudita-center-e2e/src/specs/overview/kompakt-overview.ts

```diff
@@ -61,55 +61,68 @@ describe("E2E mock sample - overview view", () => {
     )
 
     const aboutYourDevice = await OverviewKompaktPage.aboutYourDevice
-    await expect(aboutYourDevice).toBeDisplayed
+    await expect(aboutYourDevice).toBeDisplayed()
     console.log(aboutYourDevice)
     await expect(aboutYourDevice).toBeClickable
     await aboutYourDevice.click()
 
     const sarInformationButtonKompakt =
       await OverviewKompaktPage.sarInformationButtonKompakt
     const sarInformationPopup = await OverviewKompaktPage.sarInformationPopup
-    await expect(sarInformationButtonKompakt).toBeDisplayed
+    await expect(sarInformationButtonKompakt).toBeDisplayed()
     await sarInformationButtonKompakt.click()
-    await expect(sarInformationPopup).toBeDisplayed
+    await expect(sarInformationPopup).toBeDisplayed()
 
     const sarInformationPopupCloseButton =
       await OverviewKompaktPage.sarInformationPopupCloseButton
+    await sarInformationPopupCloseButton.waitForClickable()
     await sarInformationPopupCloseButton.click()
 
+    const backArrowButton = OverviewKompaktPage.backArrowButton
+    await backArrowButton.waitForClickable()
+    await backArrowButton.click()
+
     const createBackupButton = await OverviewPage.createBackupButton
-    await expect(createBackupButton).toBeDisplayed
-    await expect(createBackupButton).toBeClickable
+    await expect(createBackupButton).toBeDisplayed()
+    await expect(createBackupButton).toBeClickable()
 
     const backupInfo = await OverviewKompaktPage.backupInfo
-    await expect(backupInfo).toBeDisplayed
+    await expect(backupInfo).toBeDisplayed()
     const str: string =
       "\tYou haven’t backed up your device yet.\n Create a backup file to protect against data loss.\n"
     const backupInfoText: string = str.trim()
```

Comment:
I don't understand why you do this. 

1. Value is not used anywhere (either `str` and `backupPinfoText`) 
1. Value is instantly trimmed - if you trim your own value - why don't trim it when entering the string? 

⚠️  Do not use this code - because value is not used - that is just to show you what I mean.
```
    const backupInfoText: string =
      "You haven’t backed up your device yet Create a backup file to protect against data loss."
````

Reply:
const backupInfo = await OverviewKompaktPage.backupInfo
    await expect(backupInfo).toBeDisplayed()
    await expect(backupInfo).toHaveTextContaining(
      "You haven’t backed up your device yet.\nCreate a backup file to protect against data loss."
    )